scripts/deploy/contracts/BillBuster.ts

```diff
@@ -0,0 +1,26 @@
+import { deployContract, sleep } from "../utils";
+import { BillBuster } from "../../../build/typechain";
+import { verifyOnEtherscan } from "../../tasks/verifyContractsEtherscan";
+import hre, { ethers } from "hardhat";
+
+export const contractNames = () => ["BillBuster"];
+
+export const constructorArguments = () => [
+  "0xd9BAcC5BccAd9A380001d41Cd234b4D5f33ece76"
```

Comment:
I think this guy should get from contracts.NETWORK.REMI, so you don't have to change it every time redeploy the contract.